mlflow/tracking/fluent.py

```diff
@@ -2131,6 +2131,23 @@ def initialize_logged_model(
     return model
 
 
+@contextlib.contextmanager
+def _logged_model_cm(model: LoggedModel) -> Generator[LoggedModel, None, None]:
```

Comment:
Let's rename this to `_start_model` or something else. It's unclear what `cm` is.